Externals/expr/include/expr.h

```diff
@@ -301,6 +305,13 @@ extern "C" {
     }
   }
 
+  static std::string expr_get_str(struct expr* e) {
```

Comment:
The expr library is written to compile cleanly as C. Dolphin is C++, and the changes made here might never be upstreamed, but for consistency with the surrounding code this should probably just return the underlying null terminated string as a const char* or NULL if the type is not OP_STRING.